AngouriMath/Core/Sys/TrigonometricTableValues.cs

```diff
@@ -176,9 +176,9 @@ private static EDecimal TwoPiOver(int a)
                 (TwoPiOver(9), f1_2 * (Cbrt((-1 + Sqrt(-3)) / 2) + Cbrt((-1 - Sqrt(-3)) / 2))),
                 (TwoPiOver(10), f1_4 * (Sqrt(5) + 1)),
                 (TwoPiOver(12), f1_2 * Sqrt(3)),
-                (TwoPiOver(14), f1_24 * Sqrt(3 *
-                                          (80 + Cbrt(14336 + Sqrt(-5549064193)) + Cbrt(14336 - Sqrt(-5549064193))
-                                          ))),
+                // (TwoPiOver(14), f1_24 * Sqrt(3 * (80 + Cbrt(14336 + Sqrt(-5549064193)) + Cbrt(14336 - Sqrt(-5549064193))))),
+                // Incorrect simplification! cos(2pi/14) = 0.900968867902419126236102319507445051165919162131857150053...
+                //                  Above simplification = 0.900968867908163376042627598612270994870357666484835165523...
```

Comment:
It returned 0.9009... as well? https://www.wolframalpha.com/input/?i=1%2F24+*+Sqrt%283+*+++++++++++++++++++++++++++++++++++++++++++%2880+%2B+Cbrt%2814336+%2B+Sqrt%28-5549064193%29%29+%2B+Cbrt%2814336+-+Sqrt%28-5549064193%29%29+++++++++++++++++++++++++++++++++++++++++++%29%29%29

Reply:
See the decimals after the initial few.

Reply:
That's a question of precision then. Maybe @MomoDeve, who just pushed to AM with new Cos, fixed it

Reply:
No. Input both expressions into Wolfram Alpha and compare the decimals.

Reply:
Oh, I see. Must be another Wikipedia's mistake. Alright, I'll check it later

Reply:
And I suspected this early on because of the weirdly large `-5549064193` which stands out among the simplified values.